internal/api/handlers.go

```diff
@@ -318,6 +320,9 @@ func PostUserAccount(env *Environment) http.HandlerFunc {
 			logErrorAndWriteResponse("Username is required", http.StatusBadRequest, w)
 			return
 		}
+		if user.Password == "" {
+			user.Password, _ = GeneratePassword(8)
```

Comment:
If the generatepassword fails, maybe we can just give a default password like `password123` instead of giving a 500 server error.

Reply:
As discussed let's stick to the error and not have a hardcoded password.